### PR TITLE
updated box to warning and shortened

### PR DESCRIPTION
**Pull request by bradygaster**
updates to the signalr version differentiation box.

### CODE REVIEW

File: aspnet/includes/signalr/signalr-version-disambiguation.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-> [!NOTE]
-> This article refers to ASP.NET SignalR. If you're thinking about using SignalR to enable real-time scenarios with Java, Node.js, or in a serverless scenario, take a look at [ASP.NET Core SignalR](/aspnet/core/signalr/introduction). If you've already used ASP.NET SignalR, take a look at the [version differences](/aspnet/core/signalr/version-differences) page to understand the differences in the versions and the improvements in ASP.NET Core SignalR. Finally, if you know you'll be running your real-time apps in Microsoft Azure, take a look at the [Azure SignalR Service](/azure/azure-signalr/signalr-overview), as it provides cloud-based scale-out once your apps need it.
\ No newline at end of file
+> [!WARNING]
+> This documentation is not for the latest version of SignalR. Take a look at [ASP.NET Core SignalR](/aspnet/core/signalr/introduction) to keep up with the latest updates.
```
Comment (BrennanConroy) [inline aspnet/includes/signalr/signalr-version-disambiguation.md]: "latest updates" makes it sound like Asp.Net Core SignalR is an update of Asp.Net SignalR

**Comment by bradygaster** [comment]
adding @scottaddie for final review

File: aspnet/includes/signalr/signalr-version-disambiguation.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
-> [!NOTE]
-> This article refers to ASP.NET SignalR. If you're thinking about using SignalR to enable real-time scenarios with Java, Node.js, or in a serverless scenario, take a look at [ASP.NET Core SignalR](/aspnet/core/signalr/introduction). If you've already used ASP.NET SignalR, take a look at the [version differences](/aspnet/core/signalr/version-differences) page to understand the differences in the versions and the improvements in ASP.NET Core SignalR. Finally, if you know you'll be running your real-time apps in Microsoft Azure, take a look at the [Azure SignalR Service](/azure/azure-signalr/signalr-overview), as it provides cloud-based scale-out once your apps need it.
\ No newline at end of file
+> [!WARNING]
+> This documentation is not for the latest version of SignalR. Take a look at [ASP.NET Core SignalR](/aspnet/core/signalr/introduction).
```
Comment (scottaddie) [inline aspnet/includes/signalr/signalr-version-disambiguation.md]:
```suggestion
> This documentation isn't for the latest version of SignalR. Take a look at [ASP.NET Core SignalR](/aspnet/core/signalr/introduction).
```

**Comment by scottaddie** [comment]
Thank you, @bradygaster! I'll publish this change right now.

**Pull request merged**